src/software/ai/navigator/path_planner/hrvo/velocity_obstacle.h

```diff
@@ -18,6 +22,21 @@ class VelocityObstacle
      */
     VelocityObstacle(Vector apex, Vector side1, Vector side2);
 
+    /**
+     * Construct a velocity obstacle given the obstacle from the perspective of a robot.
+     *
+     * @param obstacle			the obstacle to consider when constructing the velocity
+     * obstacle
+     * @param robot				the robot to consider the obstacle from
+     * @param obstacle_velocity	the velocity of the obstacle
```

Comment:
nit: some of these spaces in the comments are back

Reply:
I think it's fixed now